### PR TITLE
Update to 1.2.7

### PR DESCRIPTION
**Pull request by davidh-ssec**

### CODE REVIEW

**Comment by ocefpaf** [comment]
@davidh-ssec this feedstock needs to be re-rendered with `conda-smithy`. Do you know how to do that? If not I can take care of that later.

**Comment by davidh-ssec** [comment]
@ocefpaf Done. Is there a way to know I should rerender a feedstock or should I just update conda-smithy every time and see if anything changes? Are there times that I wouldn't want to rerender?

**Comment by ocefpaf** [comment]
> @ocefpaf Done. Is there a way to know I should rerender a feedstock 

Only if you are following the developments of conda-forge closely. Which is not recommended due to the high noise level :smile: 

> or should I just update conda-smithy every time and see if anything changes?

You can rerender when the CIs fail when you know the problem is not the recipe.

> Are there times that I wouldn't want to rerender?

Nope. It is pretty safe to rerender constantly. However, in theory the bot should be ahead of the manual updates so you should not rerender at all. But the feedstock boilerplate, updates, and fixes are more frequent than what the bot can handle lately.

File: .travis.yml [hunk 1]
```diff
@@ -28,9 +28,9 @@ before_install:
 
 install:
     - |
-      MINICONDA_URL="http://repo.continuum.io/miniconda"
+      MINICONDA_URL="https://repo.continuum.io/miniconda"
```
Comment (ocefpaf) [inline .travis.yml]: @davidh-ssec this is the most important change from that re-rendering BTW. Continuum ended the `http` links and we had to move to `https`.

**Pull request merged**